etc/a.gwf

```diff
@@ -119,17 +129,19 @@ wizard_just_friend=no
 # Default is yes
 # You may define several messages, each postfixed with the two letters code for a language
 # visitor_access=no
-# visitor_access_msg_en=Access to the base is restricted to friends.
+# visitor_access_msg_en=Access to the base is restricted to wizard and friends.
 
 # Generates pages with DOCTYPE HTML 4.01 (default XHTML 1.1)
 #doctype=html-4.01
 
-# File recording a black list for the database forum.
+# File recording a black list for the database forum. (Format?? same as auth files?)
 #forum_exclude_file=
```

Comment:
Format is: one pattern per line

Patterns are just letters, with `*` meaning: any character(s).

Reply:
Ok, it is an IP blacklist, comparing with the value of conf.from with * = wild card